code/modules/mob/living/carbon/human/life/handle_fire.dm

```diff
@@ -7,3 +7,4 @@
 	if((1 - thermal_protection) > 0.0001)
 		bodytemperature += BODYTEMP_HEATING_MAX
 		adjustFireLoss(5)
+	species?.handle_fire(src)
```

Comment:
stray question mark?

Reply:
> Two new operators, ?. and ?:, have been added. These are null-conditional operators. When reading a?.b, the result will be null and will not crash the proc if a is null. When writing to a?.b, the write will simply not happen if a is null.

Reply:
Wasn't implying it was bad, i just find byond interesting.